guides/source/configuring.md

```diff
@@ -721,6 +721,8 @@ There are a number of settings available on `config.action_mailer`:
 
 * `config.action_mailer.perform_caching` specifies whether the mailer templates should perform fragment caching or not. By default this is `false` in all environments.
 
+* `config.action_mailer.delivery_job` specifies delivery job for sending parameterized and normal mail. Defaults to `ActionMailer::Parameterized::DeliveryJob` and `ActionMailer::DeliveryJob`.
```

Comment:
Does it make sense to reference `ActionMailer::Parameterized::DeliveryJob` here? I think it should just say ```specifies delivery job for sending mail. Defaults to `ActionMailer::Parameterized::DeliveryJob`.```. You can't actually specify a parameterized delivery job.

Reply:
Yeah, it doesn't make sense.

Reply:
Agreed, I just fixed it.